src/NzbDrone.Common/Disk/DiskTransferService.cs

```diff
@@ -511,6 +502,21 @@ private void TryMoveFileVerified(string sourcePath, string targetPath, long orig
             }
         }
 
+        private void VerifyFile(string sourcePath, string targetPath, long originalSize, string action)
+        {
+            var targetSize = _diskProvider.GetFileSize(targetPath);
+            if (targetSize != originalSize)
+            {
```

Comment:
New line before if block, but invert to return early if it's equal for less nesting overall.

```suggestion

            if (targetSize == originalSize)
            {
                return;
            }
```